solidity/random-beacon/contracts/RandomBeacon.sol

```diff
@@ -942,8 +1055,9 @@ contract RandomBeacon is IRandomBeacon, Ownable {
     ///         of group members, operators of members deemed as inactive are
     ///         banned for sortition pool rewards for duration specified by
     ///         `sortitionPoolRewardsBanDuration` parameter. The sender of
-    ///         the claim must be one of the claim signers. This function
-    ///         can be called only for active and non-terminated groups.
+    ///         the claim must be one of the claim signers. The sender is
+    ///         refunded from the Reimbursement Pool. This function can be
+    ///         called only for active and non-terminated groups.
```

Comment:
Should we add a similar sentence to the documentation of other functions supporting refunds? Alternatively, do _not_ add it here.

Reply:
I removed the sentence, cause other docs are smaller, and mentioning just about reimbursement pool raises a question about why we mention that and not other stuff as well.